recipes/multi-entry.md

```diff
@@ -0,0 +1,50 @@
+## 🍲 Setting up Multiple entries
+
+When building a multi-page application, in order to optimise code/module reuse across the whole application a multi entry configuration is used and developers want to use all the features webpack have, such as `hot module replacement` when one change any file. 
```

Comment:
```suggestion
Certain applications or bundles may require several separate entries. Depending on a user's needs, it may be desirable to make Hot Module Replacement, live-reload, or any other client feature available for each entries, or specific entries other than the default. To accomplish that, the `webpack-plugin-serve/client` file must be included with entries on which client features are needed. We'll show you how that should look below.
```